# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: joe@tennies.us-20110316133930-mv49d8za0xvr229w
# target_branch: http://bazaar.launchpad.net/~twisted-\
#   dev/twisted/trunk/
# testament_sha1: af5921f0578f0c50b28b99c4ca77891c7b3dbf60
# timestamp: 2011-03-16 08:42:39 -0500
# base_revision_id: svn-v4:bbbe8e31-12d6-0310-92fd-\
#   ac37d47ddeeb:trunk:31000
# 
# Begin patch
=== modified file 'INSTALL'
--- INSTALL	2010-11-23 04:09:43 +0000
+++ INSTALL	2011-03-16 13:39:30 +0000
@@ -28,3 +28,31 @@
     is:
 
       python setup.py install
+
+Supported Platforms 
+
+  Twisted takes the word "support" very seriously.  We only consider a  
+  platform supported if we have a buildbot for it.  Go to 
+  http://buildbot.twistedmatrix.com/boxes-supported to view which platforms 
+  and versions of Python currently have buildbots.   
+
+  The following platforms were considered supported at the time of release: 
+
+  * Apple OS X 10.6 
+  * Debian GNU/Linux (ia64 bit) - Lenny
+  * Microsoft Windows XP (32 bit) - requires SP2 or higher   
+  * Microsoft Windows 7 (64 bit)
+  * Ubuntu Linux (ia32 and ia64 bit) - Hardy, Lucid, 
+ 
+  The following platforms are known to have active users in the Twisted  
+  community and are therefore expected to work but did not have active  
+  buildbots available at the time of release: 
+
+  * Apple OS X (versions other than 10.6) 
+  * FreeBSD 
+  * Microsoft Windows XP (64 bit) 
+  * Microsoft Windows Vista 
+  * Microsoft Windows 7 
+  * Novell SUSE Linux Enterprise Server 
+  * OpenSUSE 
+  * Red Hat Enterprise Linux 

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWbPhlJYAAhBfgAAQUnf/93f3
38C////wUAUJq9c3HcEjohrrqEkU0TEwTU8mmpkbVNlHoph5SP1J5RmUeUbSaDJAEwiTyj1GnpED
GkYRpgaI0PQJhiEUyntQxTRpA/UepkmJ6gwQNDCMRocwJiaDCZMmTIwmCaaZGJgCGASRExBNNKem
TaT9VPRNAAaAAPU0ADI6LtR5b/U1aZZhkf8wVNEMi8eKNwMIPpxhE/SS+Xti2XU5Xzt8+CcnjBDq
1S34wIKkKIQz7yS4YdkiDcdGrFAmvxEPRGRbncjBsn6GFFPpwZk2vXcygCwI7xVPyaLAlENlI6w4
9Uzw7MkrJJtQEt+c/Bqn26MApQ74vTJQzW9eSMWIJNbnP16xG8YuE2ebVMEbqidTax4Wj2QdpnGr
lHLhP35egh0dzSETEtjAOQbdDGCeInXcDQnIlyWwFndKwGshYHbhFLLLR15+i+/l700agHFywgHI
otAwBfoRdsCXBlpTD4cJY4CoUCD+A2DWDnB2kuxknGWrvclFHDtNuVAnie+/TUQvSmLxWKs9c+jQ
L9sJ7OGoqulvKoEL4zlOKXEnF1S1YGvG2qWMTWBzIuI1WpaCicUdzb0ppQVw4oU9khYSlSpVLYHd
EoYPhkBI2DQoZ6CItWTT2BdpUAqoxgj5MgN4CbgKCtVW7wkTQnAZhYL24UVig2YzvpiNiaBJWC6q
g+lCpcCUJe++gRV7bw1MG2MlIY2EsDQUfmyJqDC0Ibd4oM4LZgaDe33umIAayuAGujNJSW0hDFSN
AlCS0JXsS5mnhPXcNufblwkUiX+LpnIjOww368oMROWjI+pRLxX1UaqZLyXi4eBQ69Ds5EXosHOj
MnRBxwWrokYMOcFGfkUQd3z2Mb+yTh3Jds5T1+8r5Bx0VWvbFHOrfNylFJzhAewTDzBjwLDifHi+
dK4817glSixmiSkoRo1hvmg7GTZL4HyJUtFGXEoPHZ1s9TdNuYuRdIO2CFSSFc2IdqE2470Wk00k
7tZqK5wchjABZdmmwlyVO9ZxXfiL8pE+vRr3LgoxgwpBnPCLcerwJGk2XJfbHshtY6UqzuqsSwu7
XcqpCtyJL+8M33Zexr2DvevalavbDZrp0tm9HJDXX4VJfSNRpga6ZmjxoihYRjSE/J8p/48ogOog
D7p1DrCGAYe3caFx3pROkofjxY8GVcnPG6MkPYXIt7SzCdAbgMqd1zsxu1KpiPtBtXDa9n3RhZ9P
wrR5AKMqZTPf8wxWYJD2B8UXXl3uDZjHaLTSxoD4fjOsUV79NpNnm5niNXZJrmGZGGsKZ+oKAdwy
7vNgcGLXUDNM+G41Tqm6myRlz4F+tzbIjEkiPQKVhJ9z7wYybVIJIIMp4gZARmgnwgEDrGCjKTpT
6OwcWMv5/UtAbETKruXh1SciWkJXBsdmQVSgrEV0tPW5dxqSqizmutqzKB1fXCxGKD4maWRbvCZw
c5xMQDFowdxv2rA4qRbRXi4nqWYX2AhQ/oIkM6MwUpksyQ1mZAZPHHaWRmJ7GSd6w0mRLLTR+lsr
ygYEMFuhaRR4RDEDhmZrIqMiIxFzBWwi1GsXnDpIpHgM2t+KWqZMraJ0XN+YQuiaS8pALwdXUeSV
CkFTk/lFLf7+oPzf050oV97ItoStskrCYWoOjQwNJGqKeZjmDQGYuJBlFmKHqNrwGRUMHmHvFonX
FmxZ2+XeK3OT/+LuSKcKEhZ8MpLA
